### ISSUE TITLE
Terraform state push with -force flag does not work

### ISSUE THREAD

**Issue by saguziel** [hashicorp/terraform#26224]
### Terraform Version

```
Terraform v0.13.2
+ provider registry.terraform.io/hashicorp/aws v3.4.0
+ provider registry.terraform.io/hashicorp/external v1.2.0
+ provider registry.terraform.io/hashicorp/kubernetes v1.12.0
+ provider registry.terraform.io/hashicorp/random v2.2.1
+ provider registry.terraform.io/hashicorp/template v2.1.2
+ provider registry.terraform.io/oktadeveloper/okta v3.4.1
+ provider registry.terraform.io/terraform-providers/okta v3.5.0

This is using the s3 backend
```

### Expected Behavior

Running `terraform state push -force` should ignore serial check.

### Actual Behavior

Running `terraform state push -force <file>` returns `Failed to write state: cannot import state with serial 70 over newer state with serial 172`, but as stated in the documentation `Both of these safety checks can be disabled with the -force flag.`

### Steps to Reproduce

1. Have existing terraform repo
2. terraform state pull > $filename
3. terraform apply (something that is NOT no-op)
4. terraform state push $filename (gets rejected due to higher serial)
5. terraform state push -force $filename (serial check should be ignored, but is not)

**Comment by saguziel** [comment]
I see https://github.com/hashicorp/terraform/pull/26190 has touched this functionality recently, but not sure if it will fix this issue.

**Comment by alisdair** [comment]
Thanks for the report! This looks like a duplicate of #25761, which as you noted was fixed in #26190. We have also back ported this to the v0.13 branch and it will be released as part of the next 0.13 point release.
